src/features/PowerShellNotebooks.ts

```diff
@@ -228,23 +291,36 @@ class PowerShellNotebookContentProvider implements vscode.NotebookContentProvide
         const retArr: string[] = [];
         for (const cell of document.cells) {
             if (cell.cellKind === vscode.CellKind.Code) {
-                retArr.push(...cell.document.getText().split(/\r|\n|\r\n/));
+                retArr.push(...cell.document.getText().split(/\r\n|\r|\n/));
```

Comment:
Is the `\r` by itself needed? Seems like a weird character to hit, but something we shouldn't care about -- if people want to embed random carriage returns in their file, should we care (given that we're not on Mac OS 8)?

Reply:
Yeah I guess we don't care about Mac OS 8 :)

Reply:
Oops there was a MacOS 9! Well that one too